R/resample-third-party-hazard-maps-to-zillow-grid.R

```diff
@@ -0,0 +1,22 @@
+# Resample the Fire risk map to be at the same resolution/extent as the Zillow data
+
+library(tidyverse)
+library(raster)
+library(rasterVis)
+
+template <- raster::raster("data/EmptyGrid/Empty_250_US.tif")
```

Comment:
This seems like a place where you could use `download_grid()` to ensure this local file exists

Reply:
okay! I don't see `download_grid()` in any packages I've loaded, with `??download_grid`, or with a Google search. Do you mean writing our own function that checks for whether the local file exists or...?

Reply:
Scratch that! I see the function in this repo (* headdesk *)